app/pb_kits/playbook/pb_collapsible/index.js

```diff
@@ -0,0 +1,82 @@
+import PbEnhancedElement from '../pb_enhanced_element'
+
+const MAIN_SELECTOR = '[data-collapsible-main]'
+const CONTENT_SELECTOR = '[data-collapsible-content]'
+const DOWN_ARROW_SELECTOR = '.fa-chevron-down'
+const UP_ARROW_SELECTOR = '.fa-chevron-up'
```

Comment:
Bonus points for doing these constants for the selectors! This makes class changes down the road much easier 💯